src/instance/InstanceService.ts

```diff
@@ -386,4 +386,22 @@ export class InstanceService {
     }
     throw new Error(`Account service for instance ${instanceId} not set.`);
   }
+
+  async sendConfirmationRead(instanceId: string, options: InstanceDeliveryOptions): Promise<string> {
```

Comment:
Where do the `InstanceDeliveryOptions` come from?

Reply:
This PR is targeting another PR.. not `dev`